FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Hai Phung on 6/15/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
```

Comment:
Following my previous comment, please remove the `internal` keyword from the `ManagedFeedImage`'s declaration, as well as from its properties and methods.
Also, this class is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```